server/athenian/api/preloading/cache.py

```diff
@@ -366,6 +373,20 @@ async def preload(self, **dbs: databases.Database) -> None:
         await gather(*tasks)
         self._log.info("Finished preloading DB tables in memory")
 
+        self._log.debug("Scheduling refresh for preloaded DB tables in memory")
+        for db_name in get_memory_cache_options().keys():
+            job = aiocron.crontab(
+                f"*/{self._refresh_frequency_minutes} * * * *",
+                func=_refresh, args=(dbs[db_name], ), start=False,
+                max_early_expiration_seconds=self._max_early_refresh_minutes)
+            job.start()
```

Comment:
1. What if we set `start=True`, we won't have to call `job.start()`.
2. We should save to `self._job` and `close()` it in our higher-level `close()`.